tests/integ/scala/test_async_job_suite.py

```diff
@@ -408,7 +407,11 @@ def test_get_query_from_async_job(session, create_async_job_from_query_id):
     else:
         async_job = df.collect_nowait()
 
-    assert async_job.query == query_text
+    with caplog.at_level(logging.DEBUG):
```

Comment:
We need to follow up with Jobs team about query history latency. cc: @sfc-gh-yixie 
I prefer we keep the `xfail` here until getting some action items from the investigation. Also what does `result` mean here? Does it refer to the query against information schema or the original query?

Reply:
After this change, if we are not able to get the query from the query history, we will assert the log but not assert the query anymore. So even if the query doesn't show up in the query history, the test will not fail. So I think it's safe to remove xfail. Hmm the log is a little misleading - result refers to the result in query_history. But since it's just a log at debug level, it should be fine